tools/ColorTool/ColorTool/Program.cs

```diff
@@ -343,6 +343,12 @@ static void PrintSchemes()
             }
         }
 
+        static void PrintSchemesDirectory()
+        {
+            string schemeDirectory = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "schemes\\");
```

Comment:
(There's still a trailing slash here!)

Reply:
Hey! Sorry for the delay on this - been super busy. The issue is all sorted.